src/LendingPool.sol

```diff
@@ -40,13 +39,29 @@ contract LendingPool is LendingPoolGuardian, Creditor, DebtToken, InterestRateMo
     address internal immutable ACCOUNT_FACTORY;
     // Contract address of the Liquidator contract.
     address internal immutable LIQUIDATOR;
+    // The unit for fixed point numbers with 4 decimals precision.
+    uint16 internal constant ONE_4 = 10_000;
     // Maximum total liquidation penalty, 4 decimal precision.
     uint256 internal constant MAX_TOTAL_PENALTY = 1100;
 
     /* //////////////////////////////////////////////////////////////
                                 STORAGE
     ////////////////////////////////////////////////////////////// */
 
+    // The current interest rate, 18 decimals precision.
+    uint256 public interestRate;
```

Comment:
make it a uint80?
Since we already cast it to a uint80

Reply:
we are casting only for event. i will change event to uint256 as well. uint80 is can be risky and its cast to uint256 for realised debt calculations